numba/tests/test_numbers.py

```diff
@@ -0,0 +1,96 @@
+# (Some) Tests for numbers.py
+from __future__ import print_function, absolute_import, division
+
+import sys
+
+import numpy as np
+
+from numba import njit
+from numba.errors import TypingError
+from .support import TestCase, unittest
+
+_32bit_only = not (sys.maxsize > 2 ** 32)
+
+
+def gen_view(a,b):
+    def impl(x):
+        return a(x).view(b)
+    return impl
+
+
+class TestViewIntFloat(TestCase):
+
+    def do_testing(self, inputs, dtypes):
+        for value, initial_type, expected in inputs:
```

Comment:
Whilst this checks that the code returns an expected value, it does not seem to execute the `non-jitted` function so as to provide a source of truth from NumPy (Numba needs to assert that it matches NumPy)?

Reply:
Luckily, since all the tests are parametrized, this is easy to add.